src/css/spectrum/spectrum.css

```diff
@@ -13,7 +13,7 @@ License: MIT
     *display: inline;
     *zoom: 1;
     /* https://github.com/bgrins/spectrum/issues/40 */
-    z-index: 9999994;
+    z-index: 20000;
```

Comment:
Was the previous value an issue ? 

If not I'd prefer not to diverge from the original spectrum css file.
If yes, you could add this custom style to the spectrum-overrides.css files. It contains all the "piskel" hacks.